### DISCUSSION TITLE
How to customize the anchor navigation

### DISCUSSION THREAD

**Discussion by junhan-z** [cloudscape-design/components#3083] [General]
Hello, I'd like to make the anchor navigation horizontal with different colors for the links, however, I didn't find how to customize it in the documentation therefore would like to ask for some help.

**Comment by cansuaa** [comment]
Hi Junhan, currently anchor navigation doesn't support horizontal view. For horizontal view, [breadcrumb-group](https://cloudscape.design/components/breadcrumb-group/) could work depending on your use case but it doesn't support different colors for links.

**Reply by junhan-z** [reply]
Thanks!
